components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/authz/OAuth2AuthzEndpoint.java

```diff
@@ -2361,7 +2361,13 @@ private String doUserAuthorization(OAuthMessage oAuthMessage, String sessionData
             return promptUserForConsent(sessionDataKeyFromLogin, oauth2Params, authenticatedUser, true, oAuthMessage);
         } else if (isPromptNone(oauth2Params)) {
             return handlePromptNone(oAuthMessage, sessionState, oauth2Params, authenticatedUser, hasUserApproved);
-        } else if (isPromptLogin(oauth2Params) || isPromptParamsNotPresent(oauth2Params)) {
+        } else if (isPromptLogin(oauth2Params) || isPromptParamsNotPresent(oauth2Params)
+                || isPromptSelectAccount(oauth2Params)) {
+            /*
+             * IS does not currently support multiple logged-in sessions.
+             * Therefore, gracefully handling prompt=select_account by mimicking the behaviour of a request that does
```

Comment:
Looks like this line is longer. Shall we break it into two lines

Reply:
The length is 118 characters which is <120 (the limit we are adhering to according to https://github.com/wso2/product-is/wiki/Contributing-to-Identity-Server#line-length)